File: TripKitAndroid/src/main/java/com/skedgo/geocoding/agregator/GCSkedGoResultInterface.java
package com.skedgo.geocoding.agregator;

import java.util.List;

import androidx.annotation.Nullable;

public interface GCSkedGoResultInterface extends GCResultInterface {

    //    skedgo result class (class json field)
    String getResultClass();

    //    skedgo result popularity (popularity json field)
    int getPopularity();

    @Nullable
    List<String> getModeIdentifiers();
}

File: TripKitAndroid/src/main/java/com/skedgo/geocoding/agregator/GCSkedGoResultInterface.kt
package com.skedgo.geocoding.agregator

interface GCSkedGoResultInterface : GCResultInterface {

    // Skedgo result class (class JSON field)
    val resultClass: String

    // Skedgo result popularity (popularity JSON field)
    val popularity: Int

    // Nullable list of mode identifiers
    val modeIdentifiers: List<String>?
}